packages/docs-app/src/examples/core-examples/buttonExamples.tsx

```diff
@@ -51,12 +65,12 @@ export const ButtonOutlinedExample: React.FC<ExampleProps> = props => {
     const code = dedent`
         <Button text="Outlined" outlined={true} />
         <Button text="Primary" outlined={true} intent="primary" />
-        <Button text="Disabled" minimal={true} disabled={true} />`;
```

Comment:
This was a typo in the current Button outlined example (unrelated to the rest of these changes)